README.md

```diff
@@ -33,7 +33,7 @@ Note that running the pipeline does _not_ automatically update the Agora databas
 into the Agora databases is handled by [agora-data-manager](https://github.com/Sage-Bionetworks/agora-data-manager/).
 
 You can run the pipeline in any of the following ways:
-1. [GitHub Actions](#GitHub-Actions) is the simplest, but least flexible, way to run the pipeline; it does not require Synapse permissions, creating a Synpase PAT, or setting up the Synapse Python client.
+1. [Nextflow Tower](#GitHub-Actions) is the simplest, but least flexible, way to run the pipeline; it does not require Synapse permissions, creating a Synpase PAT, or setting up the Synapse Python client.
```

Comment:
Update the link to the new section header: [Nextflow Tower](#GitHub-Actions) -> (#Nextflow Tower)

Reply:
Great catch @JessterB .  Thanks!